pkg/config/validation/virtualservice.go

```diff
@@ -141,6 +141,13 @@ func validateHTTPRouteMatchRequest(http *networking.HTTPRoute, routeType HTTPRou
 					if header == nil {
 						errs = appendErrors(errs, fmt.Errorf("header match %v cannot be null", name))
 					}
+
+					if _, ok := header.GetMatchType().(*networking.StringMatch_Prefix); ok {
+						if header.GetPrefix() == "" {
+							errs = appendErrors(errs, fmt.Errorf("header match %v may not be empty", name))
```

Comment:
nit:  please add prefix

Reply:
Prefix to what?

Reply:
fmt.Errorf("header prefix match %v must not be empty", name)

Reply:
Aah- done!